observability/logging/logging-6.0/log6x-upgrading-to-6.adoc

```diff
@@ -0,0 +1,387 @@
+* Removal of support for managed log storage and visualization based on Elastic products (i.e., Elasticsearch, Kibana).
+* Deprecation of the Fluentd log collector implementation.
+* Removal of support for `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` resources.
+
+[NOTE]
+====
+The *cluster-logging-operator* does not provide an automated upgrade process.
+====
+
+Given the various configurations for log collection, forwarding, and storage, no automated upgrade is provided by the *cluster-logging-operator*. This documentation assists administrators in converting existing `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` specifications to the new API. Examples of migrated `ClusterLogForwarder.observability.openshift.io` resources for common use cases are included.
+
+
+== Log Storage
+
+The only managed log storage solution available in this release is a Lokistack, managed by the *loki-operator*. This solution, previously available as the preferred alternative to the managed Elasticsearch offering, remains unchanged in its deployment process.
+// Something recently changed in the code re the Lokistack CR relating to the CLF?
+
+[IMPORTANT]
+====
+To continue using an existing Red Hat managed Elasticsearch deployment provided by the *elasticsearch-operator*, remove the owner references from the `Elasticsearch` resource named `elasticsearch` in the `openshift-logging` namespace before removing the `ClusterLogging` resource named `instance` in the same namespace.
+====
+
+== Log Visualization
+
+The OpenShift console UI plugin for log visualization has been moved to the *cluster-observability-operator* from the *cluster-logging-operator*.
+// Pending support statement.
+
+[IMPORTANT]
+====
+To continue using an existing Red Hat managed Kibana deployment provided by the *elasticsearch-operator*, remove the owner references from the `Kibana` resource named `kibana` in the `openshift-logging` namespace before removing the `ClusterLogging` resource named `instance` in the same namespace.
+====
+
+== Log Collection and Forwarding
+// Can't link to github, need to figure a workaround.
+
+Log collection and forwarding configurations are now specified under the new link:../../reference/operator/api_observability_v1.adoc[API], part of the `observability.openshift.io` API group. The following sections highlight the differences from the old API resources.
+
+[NOTE]
+====
+Vector is the only supported collector implementation.
+====
+
+== Management, Resource Allocation, and Workload Scheduling
+
+Configuration for management state (e.g., Managed, Unmanaged), resource requests and limits, tolerations, and node selection is now part of the new *ClusterLogForwarder* API.
+
+.Previous Configuration
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+spec:
+  managementState: "Managed"
+  collection:
+    resources:
+      limits: {}
```

Comment:
I'd recommend to set limits as a good practice for them. This was requested in documentation bug - https://issues.redhat.com/browse/OBSDOCS-856. The reason is because of if not limits, it can bring down a node as using all the memory as visible in https://issues.redhat.com/browse/LOG-4745. 

Then, I'd set the examples always with limits and also recommending to set/adjust to the needs of each cluster.

Also, in parallel, if https://issues.redhat.com/browse/LOG-4745 is implemented for Logging 6.0. We should indicate that if the user doesn't set limits, what these default limits are

Reply:
LOG-4745 is implemented: [code](https://github.com/openshift/cluster-logging-operator/blob/44af754c91d6a63acc04363f4b0a7345085e094e/internal/api/initialize/init_resources.go#L14-L17).